### PR TITLE
Feature b

### PR DESCRIPTION
**Pull request by ldelbel**
In this project I learned about gradients and advanced styling tools and resources, applying it in details to achieve my goal of making an apple's webpage clone. The page that was cloned is found in this link: https://web.archive.org/web/20140301004610/http://www.apple.com/ 

In this project I practiced the following topics:

- Image embedding and background image handling
- Gradient application to achieve specific visual effects
- Mixed positioning techniques to work together
- Media queries to handle responsiveness

Notes:
       - The version of the webpage that I cloned is the one I have opened in the Brave browser inside Windows. It differs in some details from the one I opened in Firefox inside Xubuntu.
       - My HTML Validator is showing a warning about Unicode Private Use Area (Charmod C073). I made some research about it, and found out there is no solution but changing all the method of including the icon to remove that warning. Because it's a warning and a non-fixable one, I chose to keep it there. Let me know if I need to change the method.

### CODE REVIEW

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @ldelbel, congratulations on the effort you put on this project :+1: Nice work :smiley: PR has a title and a description and README looks professional :smile: Let us improve our code and make it even better by addressing the comments below:
- Kindly enable stickler by visiting stickler-ci.com
- There is a horizontal scroll that needs to be fixed. Kindly check the image and see how it appears
work1
- Since this project is primarily based on using backgrounds and gradients, let us use background-images for this part of the code
work
- There are some repeating codes in our CSS file, let us apply the DRY rule. Kindly check this [article](https://speckyboy.com/good-bad-css-practices/)
- W3validator returned 1 warning, it is easily fixable, kindly address it

Kindly make these changes and submit another code review :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Comment by ldelbel** [comment]
About the warning in HTML Validator, it seems there is no simple solution for this problem. I would probably need to change my solution for inseting the icon. For more info https://stackoverflow.com/questions/23978976/fontawesome-usage-displeases-w3c-validator 

I want to know if I really need to change it, because it's a good solution in my point of view.

**Review by christianotieno — CHANGES_REQUESTED** [review]
## Hello @ldelbel 👋 
Everything looks good. There some tiny bits of changes left before we wrap this up.
- - [ ] Note that the page need not be having a scroll. A `100` for `view height` and  `view width` would suffice in this case.
- - [ ] Also note that the four elements should be links (to other articles or `#`) ([link to page](https://web.archive.org/web/20140301004610/http://www.apple.com/)).  

- - [ ] In the README, kindly add some instructions on how to best clone the project and use it, thank you.

♻️ Work on these changes and resubmit another CRR, thank you.

Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)
---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,378 @@
...
+  width: 100%;
+}
+
+.thumbs {
+  display: flex;
+  width: 100%;
+}
+
+.thumb {
+  width: 25vw;
```
Comment (rammazzoti2000) [inline styles.css]: You still have the horizontal overflow, caused by the width of .thumbs, change the width to` width: 24.7vw;`

**Review by rammazzoti2000 — APPROVED** [review]
### Hello @ldelbel you did great with the project :1st_place_medal: 
#### The project looks great. Good job for implementing the required changes.
### Project Approved :tada: :rocket: 
Before merging, to avoid a new review request, please fix the issue mentioned by the previous tse regarding the horizontal overflow.
- [ ] Change .thumb width to remove the overflow
```
.thumb {
    width: 24.7vw;
}
```
Cheers and Happy coding! :clap: 

##### Code review by Alexandru Bangau

(Feel free to contact me on slack by the handle of alexandru if you have any additional question)</em>

### :heavy_check_mark:   Status: Approved.

**Review by rammazzoti2000 — APPROVED** [review]
### :heavy_check_mark: Status: Approved

**Pull request merged**